lib/components/tiled_component.dart

```diff
@@ -69,15 +69,19 @@ class TiledComponent extends Component {
       final image = images[tile.image.source];
 
       final rect = tile.computeDrawRect();
-      final src = Rect.fromLTWH(rect.left.toDouble(), rect.top.toDouble(),
-          rect.width.toDouble(), rect.height.toDouble());
-      final dst = Rect.fromLTWH(tile.x.toDouble(), tile.y.toDouble(),
-          rect.width.toDouble(), rect.height.toDouble());
+      final src = Rect.fromLTWH(rect.left.toDouble(), rect.top.toDouble(), rect.width.toDouble(), rect.height.toDouble());
+      final dst = Rect.fromLTWH(tile.x.toDouble(), tile.y.toDouble(), rect.width.toDouble(), rect.height.toDouble());
```

Comment:
I think the only thing that changed here was the formatting right? This is what probably broke the build.

Reply:
okk.. I will see... to be honest I have to look how can I run the full build in my PC. :(

Reply:
I just looked on travis, build failed because of files formatted outside our format, the problem is probably that code section where I commented.

Even so, to format the files just run `flutter format .`